.github/workflows/mypy.yml

```
@@ -0,0 +1,43 @@
+    permissions:
+      contents: read
+    
+    name: Run tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v3
```

actionlint (1.7.7)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)